org.lflang/src/org/lflang/validation/ValidatorErrorReporter.java

```diff
@@ -0,0 +1,158 @@
+/*************
```

Comment:
I think the descendants of `ErrorReporter` are scattered too much around the project. Should they all be consolidated into one place (e.g., a package called `errorhandling`)? This makes it easier to understand what variants of `ErrorReporter` we have.

Reply:
I think they should be in different places because they are used only in very specific contexts. This one, for instance, is only useful in the validation package. The EclipseErrorReporter is only useful for Epoch and should only be in a package that already depends on eclipse. The StandaloneErroReporter is only useful for lfc and consequently located in `org.lflang.lfc`. It also depends on Kotlin, so we could not easily move this one to another package.

Reply:
I'm with @cmnrd on this one.